std/internal/math/gammafunction.d

```diff
@@ -231,7 +238,12 @@ real igammaTemmeLarge(real a, real x)
 
 public:
 /// The maximum value of x for which gamma(x) < real.infinity.
-enum real MAXGAMMA = 1755.5483429L;
+static if (real.mant_dig == 64)  // 80-bit reals
```

Comment:
I don't know about time, but DMD 2.068.

This PR can be good argument for #3011.
You may want to add dependency note in PR description: somebody from core team will add *dependency* label.